README.md

```diff
@@ -49,11 +49,8 @@ head to http://localhost:3000.
 
 Be aware of the required environment variables:
 
-- REACT_APP_MAIN_VERSION
-- REACT_APP_COGNITO_REGION
```

Comment:
`REACT_APP_COGNITO_REGION` is referenced in frontend/src/aws_cognito_region. Maybe we could have a hardcoded value?

Reply:
Sort of!  It's a default, much like for the other comments.  Our other projects should be using AWS ca-centra-1 as well.

Reply:
Right. And the only option would be replacing it locally. For deployments we'll have always ca-central-1. Got it.

Reply:
You're right!  It's unnecessary.  Thanks and removed!  :)